extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.example.1.json

```diff
@@ -11,6 +11,7 @@
   "https://ns.adobe.com/experience/journeyOrchestration/journeyVersionID": "f3485959-0a9e-4004-82d9-f11bd6c4585b",
   "https://ns.adobe.com/experience/journeyOrchestration/journeyVersionName": "Step_Event_test_2",
   "https://ns.adobe.com/experience/journeyOrchestration/origTimestamp": "2021-07-27T18:28:04.351808Z",
+  "https://ns.adobe.com/experience/journeyOrchestration/originalTimestamp": "2021-07-27T18:28:04.351808Z",
```

Comment:
Please validate if the format is correct or not, because for format : date-time, I see a example value was: `2021-01-26T06:23:28.914Z`
refer: 
https://github.com/adobe/xdm/blob/6327f347ae34c0991006185b776d68f3c378c180/extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.example.4.json#L3